File: 602/ChenRui_hw1/fourargs.cpp
// Copyright 2017 ChenRui ruirui@bu.edu
#include <iostream> 
int main(int argumentcount, char **arguments){
//        freopen("stdout.txt","w",stdout);
//        freopen("stderr.txt","w",stderr);

		for(int i =1;i<5;i++){
            if(i<argumentcount){
			std::cout<<arguments[i]<<'\n';}
        }

		for(int i=5;i<argumentcount;i++){
            if(i<argumentcount){
			std::cerr<<arguments[i]<<'\n';}
        }

    return 0;
}

File: 602/ChenRui_hw1/fourargs.py
# Copyright 2017 ChenRui ruirui@bu.edu
import sys
#fout = open('stdout_python_'+str(len(sys.argv))+'.txt','w')
#sys.stdout = fout

#ferr = open('stderr_python_'+str(len(sys.argv))+'.txt','w')
#sys.stderr = ferr

for i in range(1,5):
	if i<len(sys.argv):
	   print(sys.argv[i])
for i in range(5,len(sys.argv)):
	if i<len(sys.argv):
	   sys.stderr.write(sys.argv[i]+"\n")